labelbox/schema/batch.py

```diff
@@ -37,9 +37,10 @@ class Batch(DbObject):
     # Relationships
     created_by = Relationship.ToOne("User")
 
-    def __init__(self, client, project_id, *args, **kwargs):
+    def __init__(self, client, project_id, *args, failed_data_row_ids=None, **kwargs):
```

Comment:
This is a requirement only for Python 2.*, here is the PEP for 3.*: https://peps.python.org/pep-3102/

The intent with `failed_data_row_ids` is to make it **keyword only with default value**, if we place it before `args*` then it would fall into positional arguments section and raise `TypeError: __init__() got multiple values for argument 'failed_data_row_ids'`

Reply:
ah got it, thanks for the explanation!